ToDo/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from ToDo.views import HomeAPIView, TaskListView, TaskDetailView, TaskAddView, TaskUpdateView, TaskDeleteView
+
+urlpatterns = [
+    path('', HomeAPIView.as_view(), name='home'),
```

Comment:
there are generally two view class
Task list view and task detail view

task list view consists of  (list task and post task)
task detail view consists of (get individual task by id , update and delete)